meta/appveyor.yml

```diff
@@ -10,8 +10,8 @@
 image: Ubuntu
 
 environment:
-  TOOLCHAIN: 1.75.0
-  CARGO_GENERATE_VERSION: 0.18.3
+  TOOLCHAIN: 1.82.0
```

Comment:
Isn't that the Rust version we can't use due to reference types issues?

Reply:
Yes, but that doesn't matter for our appveyor CI since we never try to execute the actual wasm here.
Some dependency required it, but I could try again with a lower version, since `cargo-generate` is installed with `--locked` now